packages/@okta/vuepress-site/docs/reference/rate-limits/index.md

```diff
@@ -14,15 +14,15 @@ The Okta API rate limits are divided into three categories: authentication/end u
 
 To access the individual API limits, visit a category page by clicking the appropriate category link in the table.
 
-> We enforce limits at the individual API endpoint level **as requests per minute**.
+> We enforce **per-minute** limits at the individual API endpoint level.
```

Comment:
The part that I think we need to change is "at the individual API endpoint level"
Some limits are for one individual API and some are for multiple. 
Maybe we should just say "We enforce the following per-minute limits:"

Reply:
Fixed this one and the three other instances of this.